sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -937,6 +937,16 @@ object SQLConf {
       .timeConf(TimeUnit.SECONDS)
       .createWithDefault(0L)
 
+  val THRIFTSERVER_FORCE_CANCEL =
+    buildConf("spark.sql.thriftServer.forceCancel")
+      .doc("When true, all the job of query will be cancelled and running tasks will be" +
+        "interrupted. When false, all the job of query will be cancelled but running task" +
+        "will be remained until finished. Note that, this config must be set before query" +
+        "otherwise it doesn't help.")
```

Comment:
We need to describe `Note that, this config must be set before query otherwise it doesn't help.` for this config? I think the other SQL configs have the same restriction, too (users need to set a config before running a query).

Reply:
Removed.